SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularityCleaner.java

```diff
@@ -474,6 +473,11 @@ private CheckLBState checkLbState(SingularityTaskId taskId) {
       case CANCELING:
       case WAITING:
         LOG.trace("Waiting on LB cleanup request {} in state {}", loadBalancerRequestId, lbRemoveUpdate.getLoadBalancerState());
+        break;
+      case INVALID_REQUEST_NOOP:
+        exceptionNotifier.notify(String.format("LB removal failed for %s", lbAddUpdate.get().getLoadBalancerRequestId().toString()),
+          ImmutableMap.of("state", lbRemoveUpdate.getLoadBalancerState().name(), "loadBalancerRequestId", loadBalancerRequestId.toString(), "addUpdate", lbAddUpdate.get().toString()));
+        return CheckLBState.LOAD_BALANCE_FAILED;
     }
```

Comment:
only `WAITING` and `RETRY` will continue to try again. `LOAD_BALANCE_FAILED` goes to the obsolete/ignored list since the removal failed. Same end result as having it go to `DONE` except that this way we'll get a sentry telling us why it wasn't removed properly